app/components/documents/document-details-panel.js

```diff
@@ -188,6 +214,12 @@ export default class DocumentsDocumentDetailsPanel extends Component {
     } else {
       yield this.documentService.checkAndRestamp([this.args.piece]);
     }
+    
+    if (this.internalReview?.hasDirtyAttributes) {
+      yield this.internalReview.belongsTo('subcase').reload();
```

Comment:
Reloading subcase may not be needed since draft-pieces cannot be edited in this viewer (yet)
If they could be edited here in the future (I expect the request) it is possible that the submission is accepted while this view is open or being edited.
In that case we have to reload the subcase or we lose the connection to the concurrently created subcase.